### ISSUE TITLE
U+2714 and U+2717 removed?

### ISSUE THREAD

**Issue by ferenczy** [adobe-fonts/source-code-pro#172]
**Question:** Have characters `U+2714` (_Heavy check mark_) and `U+2717` (_Ballot x_) been removed from the recent version of the font?

**Explanation:** I'm using both characters in my shell prompt (the first one signals that the last command completed successfully, the later signals the opposite). Suddenly, most likely after a font update, I have just squares instead of those two characters in my shell, so I'm wondering were those characters removed from the font and if yes then why?

**Comment by jamen** [comment]
I don't have an answer, but I have a guess... Could your system (or terminal) be using a "fallback font" to supply missing Unicode characters, and these symbols never existed inside Source Code Pro in the first place?  Then maybe later on, the font that was supplying the characters got updated/removed.

I don't know the history/status of Source Code Pro, so I could be entirely wrong. 😅

**Comment by frankrolf** [comment]
U+2713 ✓ (CHECK MARK) is part of Source Code Pro, not U+2714 ✔ (HEAVY CHECK MARK).
I don’t think U+2717 has ever been part of the family.

**Comment by ferenczy** [comment]
@jamen Yep, it also came to my mind since my terminal (_ConEmu_) has such feature (called _Alternative font_), I have used it to temporarily "fix" my issue, but it was disabled before. I have backups of ConEmu's configuration files so I could check if it was enabled before. It wasn't.

**Comment by ferenczy** [comment]
@frankrolf I have just checked an older version `2.010R` and it's the same.

So I really don't know what happened on my system. I didn't change anything and suddenly I had just squares instead of characters `U+2714` and `U+2717`.

May I ask font maintainer(s), would it be possible to implement those characters, ideally from `U+2713` to `U+2718`? I think it's quite useful for developer related stuff.

**Comment by kenlunde** [comment]
This is precisely why font fall back is a double-edge sword. On one edge is the advantage of displaying arbitrary characters. On the other edge is not really knowing with what font a particular character is being rendered. Obviously something happened with font fallback, because the glyph for the characters in question were never included in this typeface family.

**Comment by frankrolf** [comment]
Those characters are part of Zapf Dingbats, which is one of the most common dingbat fonts. Your OS should come with a font that at least supports the Unicode range (AFAIK, most OSes do) – perhaps you did some font purging?

**Comment by ferenczy** [comment]
Guys, I wrote I didn't have the _font fallback_ enabled for my terminal before. I don't know if operating system (Windows 7) has anything like that, I haven't explicitly use anything like that. I'm just trying to find out how is it possible it has been working before (without the _font fallback_) and why it stopped. I have fonts which implement those characters, still I think it would be useful for _Source Code Pro_ to have such characters also.

**Comment by ferenczy** [comment]
So to keep this issue relevant, I would change it into a feature request: would it be possible to implement characters from `U+2714` to `U+2718`, please? (`U+2713` is already implemented)
